webapp/src/apis/api-helper.ts

```diff
@@ -26,17 +38,38 @@ class ApiHelper {
           Authorization: token,
         },
       })
-      console.debug('GOT RESPONSE FROM', url + url_ext, resp)
 
-      return await resp.json()
+      const response = await resp.json()
+      console.debug(
```

Comment:
Should we pull these debug lines across the app? Maybe a separate PR to manage these better...and a logging module that can be turned on/off to actually log the messages out?

Reply:
I like this idea. Just to confirm, do you want me to remove these log messages? I kinda like them right now because I can switch the log level in the console to verify which requests are being made

Reply:
I think we can leave them in for now, and we'll adjust with logging module in the future